recipes/eglibc/eglibc.inc

```diff
@@ -24,14 +24,18 @@ LIBTOOL_DEPENDS = ""
 
 MACHINE[unexport]       = "1"
 
-RECIPE_FLAGS:native = "toolchain_kernel_version_native"
-RECIPE_FLAGS:machine = "toolchain_kernel_version_machine"
-RECIPE_FLAGS:sdk = "toolchain_kernel_version_sdk"
+RECIPE_FLAGS:>native	= " toolchain_kernel_version_native eglibc_obsolete_rpc"
+RECIPE_FLAGS:>machine	= " toolchain_kernel_version_machine eglibc_obsolete_rpc"
+RECIPE_FLAGS:>sdk	= " toolchain_kernel_version_sdk eglibc_obsolete_rpc"
 
 DEFAULT_USE_toolchain_kernel_version_native	= "2.6.32"
 DEFAULT_USE_toolchain_kernel_version_machine	= "2.6.32"
 DEFAULT_USE_toolchain_kernel_version_sdk	= "2.6.32"
 
+OBSOLETE_RPC = ""
+DEFAULT_USE_eglibc_obsolete_rpc = "1"
+OBSOLETE_RPC:>USE_eglibc_obsolete_rpc = " --enable-obsolete-rpc"
```

Comment:
You could just have used OBSOLETE_RPC:USE_eglibc_obsolete_rpc but the result would have been the same.